helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -118,6 +118,23 @@ controller:
     serviceMonitor:
       enabled: true
 
+  {{- if .Values.ingressNginx.controller.chroot }}
+  image:
+    chroot: true
+  containerSecurityContext:
+    seccompProfile:
+      type: RuntimeDefault
+    capabilities:
+      drop:
+      - ALL
+      add:
+      - NET_BIND_SERVICE
+      - SYS_CHROOT
+      - SYS_ADMIN
```

Comment:
The [announcement blog post](https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/) doesn't mention needing `SYS_ADMIN`. What causes it to be needed?

Reply:
Running chroot requires the `unshare` syscall, which requires the `SYS_ADMIN` capability. The pod isn't able to start without it.